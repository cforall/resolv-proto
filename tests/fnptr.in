f-noret 1
1 f-noargs
1 f-both 1
1 &f-var

noret 1 [ : 1 ]
1 noargs [ 1 : ]
1 both 1 [ 1 : 1 ]
var 1

%%

noret( 1 &f )
noargs( &f )
both( 1 &f )
var( &f )