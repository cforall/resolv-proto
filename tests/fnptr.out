
ERROR: ambiguous resolution for noret( 1 &f )

[1 / (0,0,0,0,0)] noargs( &f-noargs )

[1 / (0,0,0,0,0)] both( 1 &f-both )

[Void / (0,0,0,0,0)] var( &f-var )
